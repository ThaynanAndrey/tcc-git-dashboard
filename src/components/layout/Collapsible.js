import React from 'react';

/**
 * Stateless component with the function of presenting a Collapsible component.
 * 
 * @author Thaynan Nunes
 */
const Collapsible = ({ children }) => (
    <ul className="collapsible">
        { children }
    </ul>
);

export default Collapsible;