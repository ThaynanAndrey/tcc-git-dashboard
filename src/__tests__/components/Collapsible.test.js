import React from 'react';
import renderer from 'react-test-renderer';

import Collapsible from '../../components/layout/Collapsible';

it('renders correctly collapsible component', () => {
    const tree = renderer.create(
        <Collapsible />
        ).toJSON();

    expect(tree).toMatchSnapshot();
});