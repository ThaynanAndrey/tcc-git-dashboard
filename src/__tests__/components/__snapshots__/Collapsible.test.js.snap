// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly collapsible component 1`] = `
<ul
  className="collapsible"
/>
`;
